src/lib/journey/stages/webauthn.svelte

```diff
@@ -92,10 +104,11 @@
     </div>
   {/if}
 
+  {#if hasDeviceName}
```

Comment:
Okay, I think I know what you're doing here, but it's not obvious to me. Tell me if this is right:

If `hasDeviceName` is `false`, you don't want to show the spinner as you're waiting for the user to input the name or skip naming it, yeah? Then, when the user provides a name, you now have a device name, so we show spinner as we wait for the WebAuthn API to complete.

Then, below, if we don't have device name, we render the device naming UI.

If I have the above right, can we improve the naming here? I'm guessing you want to use the same "flag" for both conditions, but it does read a bit weird. Rather, could we use different variables, rather than trying to combine them into one? So, for the device naming, we use as variable called `requestsDeviceName` and the Spinner UI uses `waitingForWebAuthnAPI`.

This way, the condition relates directly to the `true` condition, rather than being inversed. Does that make sense?

Reply:
I used the above names in the code, let me know if thats satisfactory